src/main/java/dev/dirs/Util.java

```diff
@@ -183,27 +194,33 @@ private static void initBase64Encoding() {
     }
   }
 
-  private static String[] runCommands(int expectedResultLines, Charset charset, String... commands) {
-    final ProcessBuilder processBuilder = new ProcessBuilder(commands);
-    Process process;
-    try {
-      process = processBuilder.start();
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    }
+  private static Process makeProcessForWindows(String ...commands) throws IOException {
+    final String[] fullCommand = new String[commands.length + 2];
+    fullCommand[0] = "cmd.exe";
```

Comment:
Is the indirection through `cmd.exe` necessary because of the change in `PATH` below?

Reply:
Yes.